tools/eslint-plugin-cdk/lib/rules/no-core-construct.ts

```diff
@@ -9,23 +9,19 @@ const BANNED_TYPES = [ 'IConstruct', 'Construct' ];
 export function create(context: Rule.RuleContext): Rule.NodeListener {
   return {
 
-    // `node` is a type from @typescript-eslint/typescript-estree, but using 'any' for now
-    // since it's incompatible with eslint.Rule namespace. Waiting for better compatibility in
-    // https://github.com/typescript-eslint/typescript-eslint/tree/1765a178e456b152bd48192eb5db7e8541e2adf2/packages/experimental-utils#note
-    // Meanwhile, use a debugger to explore the AST node.
```

Comment:
minor: Would prefer to keep this line as a note to future developers on how to work on this linter (using the debugger). This is how I did it at least.

Reply:
done